src/libsync/propagateupload.cpp

```diff
@@ -268,6 +270,8 @@ void PropagateUploadFileQNAM::slotComputeTransmissionChecksum(const QByteArray&
 
     connect(computeChecksum, SIGNAL(done(QByteArray,QByteArray)),
             SLOT(slotStartUpload(QByteArray,QByteArray)));
+    connect(computeChecksum, SIGNAL(done(QByteArray,QByteArray)),
+            computeChecksum, SLOT(deleteLater()));
     const QString filePath = _propagator->getFilePath(_item->_file);
```

Comment:
Yes. But note that the instances are parented to the propagation jobs, so it shouldn't be a big deal.